external/easylogging++/easylogging++.cc

```diff
@@ -1979,6 +1979,7 @@ void VRegistry::setLevel(base::type::VerboseLevel level) {
     m_level = base::consts::kMaxVerboseLevel;
   else
     m_level = level;
+  m_level = level;
```

Comment:
?

Reply:
Likely a first try that I did not fully undo. Fixed.